pycozmo/behavior.py

```diff
@@ -44,21 +177,75 @@ def __init__(self,
                  behavior_id: str,
                  needs_action_id: str,
                  display_name_key,
-                 params: Dict[str, Any]) -> None:
+                 params: BehaviorParameters) -> None:
         self.behavior_class = str(behavior_class)
         self.id = str(behavior_id)
         self.needs_action_id = str(needs_action_id)
         self.display_name_key = str(display_name_key)
-        self.params = dict(params)
+        self.params = params
+
+    @classmethod
+    def from_json(cls, data: Dict):
+        return cls(behavior_class=data['behaviorClass'],
+                   behavior_id=data['behaviorID'],
+                   needs_action_id=data.get('needsActionID', ''),
+                   display_name_key=data.get('displayNameKey', ''),
+                   params=BehaviorParameters.from_json(data.get('params')) if 'params' in data else None)
 
 
-def load_behaviors() -> Dict[str, Behavior]:
-    # TODO: Load cozmo_resources/config/engine/behaviorSystem/behaviors/*/*.json
+class TestBehavior:
+    def __init__(self,
+                 behavior_class: str,
+                 behavior_id: str,
+                 loop_forever: Optional[bool] = False,
+                 test_gap: Optional[float] = 0.0,
+                 runs_per_test: Optional[int] = 1) -> None:
+        self.behavior_class = str(behavior_class)
+        self.id = str(behavior_id)
+        self.loop_forever = loop_forever
+        self.test_gap = test_gap
+        self.runs_per_test = runs_per_test
```

Comment:
If `Optional` is needed this could be:
```
self.runs_per_test = int(runs_per_test) if runs_per_test is not None else None
```

Reply:
But wouldn't it be better to initialize it to 1? Then if any time some test need to be run, this variable would most likely be used to count the times a test runs on a loop. Saying the default is 1 would skip some if-else statements, maybe?

I don't know, I'm happy to change it to how you like it best. It will be some time before any logic comes in this class anyways.

Let me know what you prefer :)

Reply:
If 1 makes more sense for a default value then
```
self.runs_per_test = int(runs_per_test) if runs_per_test is not None else 1
```

Reply:
Should I do something similar for the other optionals?